newIDE/app/src/ResourcesList/index.js

```diff
@@ -155,6 +155,25 @@ export default class ResourcesList extends React.Component<Props, State> {
     );
   };
 
+  _getResourceThumbnail = (resource: gdResource) => {
+    switch (resource.getKind()) {
+      case 'image':
+        return getLocalResourceFullPath(this.props.project, resource.getName());
```

Comment:
The name is not necessarily the file path. Use getFile

Reply:
This is using getLocalResourceFullPath - is this electron only?
What is happening in the web app when this is called?